misc/openapi-ballerina/modules/openapi-validator/src/main/java/org/ballerinalang/openapi/validator/Constants.java

```diff
@@ -40,4 +40,6 @@
     static final String OPERATIONS = "operations";
     static final String BODY = "body";
     static final String FAILONERRORS = "failOnErrors";
+    static final String EXCLUDETAGS = "excludeTags";
```

Comment:
Just wondering where we made these access modifiers as default intentionally? 
Usually it has to be public.

Reply:
not intensional i think... we can make them public as it cause no harm

Reply:
@lnash94 can you make these public.... follow the below example
```suggestion
    public static final String EXCLUDETAGS = "excludeTags";
```